epsagon_kubernetes_subscription.sh

```diff
@@ -0,0 +1,105 @@
+#! /bin/bash
+
+## Script to attach Epsagon Role to kubernetes
+
+function usage {
+    echo "Usage: epsagon_kubernetes_subscription.sh EPSAGON_TOKEN"
+}
+
+ROLE_URL=https://raw.githubusercontent.com/epsagon/epsagon-k8s-role/master/epsagon-role.yaml
+
+function fetch_epsagon_role {
+    echo "Fetching epsagon-role.yaml"
+    if [ -f 'epsagon-role.yaml' ] ; then
+        echo "epsagon-role.yaml already exists - using that file"
```

Comment:
Is that a good idea? If we make a change, a customer will have to delete the existing file in order to get the change. 
I think there's no need to cache, they're not expected to use it more than once or twice.

Reply:
So actually it doesn't matter because these commands download will not overwrite anyway. We can add a question but again - I want to keep this simple.